src/main/java/com/hu/webdriver/pages/profilepage/ProfilePage.java

```diff
@@ -0,0 +1,333 @@
+package com.hu.webdriver.pages.profilepage;
+
+import java.util.Calendar;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import com.hu.webdriver.pages.BasePage;
+
+/**
+ *
+ * @author SHIVA
+ *
+ */
+public class ProfilePage extends BasePage {
```

Comment:
I am not reviewing this file, split this file into two pages
1)ProfilePage
2)EditProfilePage.